src/pizzeria/Order.java

```diff
@@ -0,0 +1,84 @@
+package pizzeria;
+
+import java.time.LocalTime;
+import java.util.*;
+
+public class Order {
+    private final int orderNumber;
+    private final int customerNumber;
+    private final List<String> pizzaName;
+    private final String pizzaType;
+    private final List<Integer> quantityOfOrder;
+    private final LocalTime time = LocalTime.now();
+    private final Set<Ingredients> getFilledPizza = new HashSet<>();
+
+    Order(int customerNumber, List<String> pizzaName, List<Integer> quantityOfOrder, String pizzaType) {
+        Random order = new Random();
+        this.orderNumber = order.nextInt(100000);
+        this.customerNumber = customerNumber;
+        this.pizzaName = pizzaName;
+        this.quantityOfOrder = quantityOfOrder;
+        this.pizzaType = pizzaType;
+    }
+
+    public int getOrderNumber() {
+        return orderNumber;
+    }
+
+    public Set<Ingredients> getFillPizza() {
+        return getFilledPizza;
+    }
+
+    public int getCustomerNumber() {
+        return customerNumber;
+    }
+
+    public List<String> getPizzaName() {
```

Comment:
This validation is necessary inside Pizza class, when you create a new Pizza object.

Reply:
done